sources/httpUtils.ts

```diff
@@ -33,6 +33,19 @@ export async function fetch(input: string | URL, init?: RequestInit) {
       dispatcher: agent,
     });
   } catch (error) {
+    if (error?.message?.includes?.(`Request cannot be constructed from a URL that includes credentials`)) {
+      const url = new URL(input as string);
+      const authorization = `Bearer ${Buffer.from(`${url.username}:${url.password}`).toString(`base64`)}`;
+      url.username = ``;
+      url.password = ``;
+      return fetch(url, {
+        ...init,
+        headers: {
+          ...init?.headers,
+          authorization,
+        },
+      });
+    }
```

Comment:
We should probably check if a username/password is provided and do this before an error is thrown.